jira/utils/__init__.py

```diff
@@ -77,8 +76,10 @@ def threaded_requests(requests):
 
 def json_loads(r):
     raise_on_error(r)
-    if len(r.text):  # r.status_code != 204:
-        return json.loads(r.text)
-    else:
+    try:
+        return r.json()
```

Comment:
@ssbarnea I'd very much like to keep this change / make this change as well. If it needs to go in another PR please let me know :-).